keymanapp/keyman
# chore(common): Check in crowdin strings for Kanuri 📜

This checks in the crowdin strings for the locale `kr` for Kanuri. Does not include macOS.

- [x] TODO: @sgschantz to check on adding kr (kr-NG?) locale in XCode

## User Testing
For the listed platforms:
1. Load the PR build
2. Set the UI/locale to Kanuri
3. Verify the UI is Kanuri
Note: The crowdin translator left some strings in English, so that's out of our control

* **TEST_ANDROID**
1. Verify Kanuri strings are like the screenshot
![Screenshot_1639536811](https://user-images.githubusercontent.com/7358010/146115662-4d10b7d2-def8-40b8-822f-aaaae1b7ae4b.png)

* **TEST_IOS** - Wait until Shawn addresses the XCode TODO before testing

* **TEST_LINUX**
1. start Keyman configuration with: `LANGUAGE=kr_NG km-config`

* **TEST_WINDOWS**
1. start Keyman
2. Change UI to "Nasara" (that's the language name the TWB translator used)
